Form/Type/DocumentType.php

```diff
@@ -60,13 +60,9 @@ public function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('document_manager', ['null', 'string', DocumentManager::class]);
     }
 
-    /**
-     * @internal Symfony 2.8 compatibility
-     *
-     * @return string
-     */
+    /** @internal Symfony 2.8 compatibility */
     #[ReturnTypeWillChange]
```

Comment:
Since the return type have been modified, this attribute is no longer necessary.